crates/bindings-macro/src/table.rs

```diff
@@ -1,12 +1,14 @@
 use crate::sats;
 use crate::sym;
 use crate::util::{check_duplicate, check_duplicate_msg, ident_to_litstr, match_meta};
+use core::slice::from_ref;
```

Comment:
I'd say probably just import `slice` and call `slice::from_ref()`; `from_ref()` as a free identifier doesn't give much context.